$NetBSD: patch-configure,v 1.1 2015/07/27 22:24:51 wiz Exp $

Use xft-config to find proper linker flags for xft.

--- configure.orig	2011-11-05 14:29:03.000000000 +0000
+++ configure
@@ -7494,7 +7494,8 @@ if ${ac_cv_lib_Xft_XftInit+:} false; the
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lXft  $LIBS"
+LIBS="`xft-config --libs`  $LIBS"
+CFLAGS="$CFLAGS `xft-config --cflags`"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
